libs/template/src/lib/push/push.pipe.ts

```diff
@@ -84,13 +84,13 @@ export class PushPipe<U> implements PipeTransform, OnDestroy {
     const scope = (cdRef as any).context;
     this.subscription = this.templateObserver.values$
       .pipe(
-        filter<RxNotification<U>>(
+        filter<RxNotification<any>>(
```

Comment:
I think u can drop the whole type specification if we go with `any` .
```suggestion
        filter(
```

Reply:
Agreed.